modin/experimental/core/storage_formats/hdk/query_compiler.py

```diff
@@ -687,7 +691,7 @@ def reset_index(self, **kwargs):
             self._modin_frame.reset_index(drop), shape_hint=shape_hint
         )
 
-    def astype(self, col_dtypes, **kwargs):
+    def astype(self, col_dtypes, errors: str):
```

Comment:
Why has it changed?

Reply:
The way the new client query compiler converts types, we can't tell whether things will error until we actually try to cast types within the query compiler. So the query compiler has to know what do with errors. The other query compilers, including base, pandas, and HDK, can ignore the `errors`. I did the same with `drop`.

Reply:
I would rather one would extract this to a separate PR